lang/java/reef-webserver/src/main/java/org/apache/reef/webserver/HttpServerReefEventHandler.java

```diff
@@ -280,6 +280,8 @@ private void writeEvaluatorInfoWebOutput(
         writer.write("<br/>");
         writer.println("Evaluator Type: " + evaluatorDescriptor.getProcess());
         writer.write("<br/>");
+        writer.println("Evaluator Runtime Name: " + evaluatorDescriptor.getRuntimeName());
```

Comment:
You should also change the Avro schema file such that the JSON serialization also contains Evaluator Runtime Name.